mesh/pncmesh.hpp

```diff
@@ -70,7 +70,8 @@ class ParNCMesh : public NCMesh
    ParNCMesh(MPI_Comm comm, const NCMesh& ncmesh, int* part = NULL);
 
    /// Load from a stream. The id header is assumed to have been read already.
-   ParNCMesh(MPI_Comm comm, std::istream &input, int version, int &curved);
+   ParNCMesh(MPI_Comm comm, std::istream &input,
```

Comment:
Same as above: As an aid to other developers we should probably take this opportunity to document these arguments. What values of `version` are recognized, are `curved` and `is_nc` treated as booleans?